cachito/workers/tasks/general.py

```diff
@@ -171,11 +172,26 @@ def save_bundle_archive_checksum(request_id: int) -> None:
 
 @app.task(priority=10)
 @runs_if_request_in_progress
-def finalize_request(request_id):
-    """Execute tasks to finalize the request creation."""
+def process_fetched_sources(request_id):
```

Comment:
I'm not sure about giving this priority 10, in https://github.com/release-engineering/cachito/commit/75f0b82639383449da19f93a3bb1ae5cee8f47be, the intention seems to have been to give the final task a higher priority then all the other tasks. I don't think it was necessary though, this is probably fine :thinking:

Reply:
Should we remove the priority on the new task then? I think this is not necessary, since we are using chained tasks, and they have a strict execution order.

Reply:
As long as the first task in the chain is lower prio than everything else then it doesn't matter AFAIK. So I don't mind either way, maybe remove the priority and let it have the default 5 for consistency